CHANGES.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog
 
+### 2.5.0
+ * Update: transitioned from Coinbase Pro (retired) to Coinbase Advanced Trade
```

Comment:
the next version is 2.4.2

Reply:
sorry, 2.4.1. Can you move this line under the existing 2.4.1 release notes?

Reply:
done